src/main/java/org/opensearch/forecast/constant/ForecastCommonMessages.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.forecast.constant;
+
+import static org.opensearch.forecast.constant.ForecastCommonName.CUSTOM_RESULT_INDEX_PREFIX;
+
+public class ForecastCommonMessages {
+    // ======================================
+    // Validation message
+    // ======================================
+    public static String INVALID_FORECAST_INTERVAL = "Forecast interval must be a positive integer";
+    public static String NULL_FORECAST_INTERVAL = "Forecast interval should be set";
+    public static String INVALID_FORECASTER_NAME =
+        "Valid characters for forecaster name are a-z, A-Z, 0-9, -(hyphen), _(underscore) and .(period)";
+
+    // ======================================
+    // Resource constraints
+    // ======================================
+    public static final String DISABLED_ERR_MSG = "Forecast functionality is disabled. To enable update plugins.forecast.enabled to true";
+
+    // ======================================
+    // RESTful API
+    // ======================================
+    public static String FAIL_TO_CREATE_FORECASTER = "Fail to create forecaster";
+    public static String FAIL_TO_UPDATE_FORECASTER = "Fail to update forecaster";
+    public static String FAIL_TO_FIND_FORECASTER_MSG = "Can't find forecaster with id: ";
```

Comment:
Nit: maybe use "Can not..." -- if OpenSearch is translated using wrappers, one can see the English abbreviations being one more point of worry.

Reply:
yes, changed